kuma/search/managers.py

```diff
@@ -12,6 +12,10 @@ class FilterManager(models.Manager):
     def visible_only(self):
         return self.filter(visible=True)
 
+    @lru_cache(maxsize=None)
```

Comment:
I started there but I didn't like the `property`ness of a manager call. I didn't look into the `maxsize` thing much but did see we are using `lru_cache` in other places. I did want this cached since it'll likely rarely change and it's a small set of data. Something like `[('topic', 'css'), ('topic', 'html'), ('topic', 'js')]`.